manifest.php

```diff
@@ -235,4 +238,7 @@
     'containerServiceProviders' => [
         CopierServiceProvider::class,
     ],
+    'includedRoles' => [
+        TaoItemsRoles::ITEM_AUTHOR => [TaoItemsRoles::ITEM_CONTENT_CREATOR]
```

Comment:
Lets also include the other roles related to ItemAuthor here (if any). We can check the mapping provided by business or ask them.

Reply:
new roles from editing items on https://www.taotesting.com/user-guide/managing-tao/management/managing-tao-management-atomic-roles/ have been added, thanks!